diff --git a/node_modules/@vscode/debugprotocol/lib/debugProtocol.d.ts b/node_modules/@vscode/debugprotocol/lib/debugProtocol.d.ts
index 4204bc2..4013f23 100644
--- a/node_modules/@vscode/debugprotocol/lib/debugProtocol.d.ts
+++ b/node_modules/@vscode/debugprotocol/lib/debugProtocol.d.ts
@@ -6,7 +6,7 @@
 /** Declaration module describing the VS Code debug protocol.
 	Auto-generated from json schema. Do not edit manually.
 */
-export declare module DebugProtocol {
+export declare namespace DebugProtocol {
 
 	/** Base class of requests, responses, and events. */
 	interface ProtocolMessage {
